examples/testdata/wasi_args.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+)
+
+//go:wasm-module wasi_snapshot_preview1
+//export args_get
+func args_get(uint32, uint32) uint16
+
+//go:wasm-module wasi_snapshot_preview1
+//export args_sizes_get
+func args_sizes_get(uint32, uint32) uint16
+func main() {
```

Comment:
I think what most confused me are things that look like unit tests here. since this is an example, how about just printing out the arg count and values. and verifying that in the unit test?

Reply:
Moved from the example directory.